dev.sh

```diff
@@ -6,4 +6,4 @@ set -e
 
 LOOTBOX_PORT="${LOOTBOX_PORT:-7191}"
 
```

Comment:
- In deploy/engine.service lootbos should be changed to engineapi
https://github.com/bugout-dev/engine/blob/991c9a545e8c64239b1dd7c1bf34552f322d311a/deploy/engine.service#L14

- Everything renamed in setup.py
https://github.com/bugout-dev/engine/blob/991c9a545e8c64239b1dd7c1bf34552f322d311a/setup.py#L11

Reply:
And if you started changing dev.sh, so  pls change to the end LOOTBOX_PORT, etc :)